bears/general/TextLintBear.py

```diff
@@ -0,0 +1,194 @@
+                    NpmRequirement('textlint-rule-period-in-list-item',
+                                   '0.2.0'),
+                    NpmRequirement('textlint-rule-rousseau', '1.4.5'),
+                    NpmRequirement('textlint-rule-unexpanded-acronym',
+                                   '1.2.1'),
+                    NpmRequirement('textlint-rule-write-good', '1.6.0'),
+                    PipRequirement('docutils-ast-writer', '0.1.2')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Grammar', 'Spelling'}
+    SEE_MORE = 'https://github.com/textlint/textlint'
+
+    @staticmethod
+    def generate_config(filename, file,
+                        check_todos: bool=None,
+                        dont_start_with_duplicated_conjunction: bool=True,
+                        no_empty_section: bool=True,
+                        check_date_weekday_mismatch: bool=True,
+                        check_grammar: bool=True,
+                        max_lines_per_file: int=300,
+                        max_comma_per_sentence: int=4,
+                        check_no_good_words: typed_list(str)=[],
+                        period_in_list_item: bool=True,
+                        minimum_acronym_length: int=3,
+                        maximum_acronym_length: int=5,
+                        ignore_acronyms: typed_list(str)=[],
+                        check_rousseau: bool=True,
+                        check_offensive_expressions: bool=True,
+                        check_common_misspellings: bool=True,
+                        write_good: bool=True,
+                        check_relative_links: bool=False,
+                        base_uri: str='null',
+                        ignore_links: typed_list(str)=[],
+                        textlint_config: str=''):
+        """
+        :param check_todos:
+            This rule checks for occurrences of ``- [ ]``
+            (so called task lists).
+        :param dont_start_with_duplicated_conjunction:
+            This rule checks whether your sentence starts with a duplicated
+            conjunction.
+        :param no_empty_section:
+            This rule does not allow to create an empty section.
+            For example, there is an empty section ``# Header A`` below:
+            ```
+            # Header A
+
+            # Header B
+
+            Text.
+            ```
+        :param check_date_weekday_mismatch:
+            This rule finds a mismatch between a date and the corresponding
+            weekday.
+        :param check_grammar:
+            This rule checks your English grammar with Ginger Proofreading.
+        :param max_lines_per_file:
+            Number of lines allowed per file.
+        :param max_comma_per_sentence:
+            Number of commas allowed per sentence.
+        :param check_no_good_words:
+            Set of NG (No Good) words to check for.
```

Comment:
hm maybe we should rename the setting to just `no_good_words`, as the prefix `check_` suggests that this is a boolean setting. What do you think?